.github/ci/after_make.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh -l
+
+set -x
+
+# Install (needed for some tests)
+make install
```

Comment:
Doesn't have to be addressed here, but I think this is resolved in the bazel branch, I'll port it over.

Reply:
Oh sweet, several libraries need this fix

Reply:
Hoping to do away with it across the stack :crossed_fingers: